ui/changes/src/graph.js

```diff
@@ -21,80 +21,36 @@ function generateData(count, yrange) {
   return series;
 }
 
-function rerender(connections, teamComponentMapping) {
+async function generateData() {
```

Comment:
Are you planning on implementing the additional features (like team-based vs component-based) in follow-up PRs?

Reply:
Yes!